src/components/Main.tsx

```diff
@@ -462,7 +462,7 @@ const EdgeAppStack = () => {
         name="exchangeSettings"
         component={SwapSettingsScene}
         options={{
-          headerTitle: lstrings.settings_exchange_settings,
+          title: lstrings.settings_exchange_settings,
```

Comment:
This is a fixup for the previous commit, where this was changed to `headerTitle` in the first place. So, please squash this commit away.